vignettes/notame_normalization_and_feature_selection.qmd

```diff
@@ -0,0 +1,315 @@
+---
+title: "Normalization and feature selection with the notame package"
+format: html
+tbl-cap-location: bottom
+editor: visual
+minimal: true
+bibliography: notame-references.bib
+date: 'Compiled: `r format(Sys.Date(), "%B %d, %Y")`'
+vignette: >
+  %\VignetteIndexEntry{Complete end-to-end LC-MS/MS Metabolomic Data analysis}
+  %\VignetteEngine{quarto::html}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+library(knitr)
+library(quarto)
+knitr::opts_knit$set(root.dir = './')
+```
+
+# Introduction
+Herein, we will perform data pretreatment and feature selection using the *notame* R package, which was developed in parallel with the associated protocol article published in the "Metabolomics Data Processing and Data Analysis—Current Best Practices” special issue of the Metabolites journal [@klavus_notame_2020]. The main outcome is identifying interesting features for laborious downstream steps relating to biological context, such as annotation and pathway analysis, which fall outside the purview of *notame*. The associated protocol article presents a sequence of steps that is easily adopted by new practitioners. We will not follow the protocol article exactly and instead do an analysis which meshes with the trunk of the workflow.
+
+# Setup
+Let's attach the *notame* package and load the preprocessed data as a SummarizedExperiment object returned from MSExperiment::quantify().
+
+```{r, message=FALSE, warning=FALSE}
+library(alabaster.se) # Portable loading and saving of SummarizedExperiment
+library(doParallel) # Parallelization
+library(cowplot) # Combinine plots in pane
+library(limma) # Differential expression analysis
+library(notame) # The main untargeted LC-MS metabolomics packages used herein
+library(SummarizedExperiment) # The SummarizedExperiment container
+
+registerDoParallel(cores = 14)
+
+dir <- system.file("extdata", "preprocessed_res", package = "metabonaut")
```

Comment:
Hi sorry just tiny thing that needs to change: can you write `Metabonaut` with a capital letter instead of `metabonaut`